.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "yaml.schemas": {
+        "https://json.schemastore.org/yamllint.json": "file:///home/will/code/architecture-as-code/spectral/instantiation/validation-rules.yaml"
```

Comment:
It looks like this one probably should not get checked in? Do we have a .gitignore for this?